pkg/bindinfo/binding_cache.go

```diff
@@ -52,26 +55,42 @@ type fuzzyBindingCache struct {
 	fuzzy2SQLDigests map[string][]string // fuzzyDigest --> sqlDigests
 
 	sql2FuzzyDigest map[string]string // sqlDigest --> fuzzyDigest
+
+	sync func(string) (Bindings, error)
```

Comment:
The name `sync` seems too general, please use a more specified name like `loadBindingFromStorage` or something like that.
And please name the input and output arguments, for example, should we use `fuzzyDigest` or `sqlDigest` as the input of this function?

Reply:
I have updated it.